src/Conversion/ONNXToTOSA/Flow/EntryPoint.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===---------------- EntryPoint.cpp - EntryPoint Op --------------------===//
+//
+// Copyright (c) 2022 Advanced Micro Devices, Inc.
+//
+// =============================================================================
+//
+// This file lowers ONNX EnryPoint to TOSA dialect.
```

Comment:
This boilerplate text does not match what it really does. Could to please write a bit more to explain what the intent is?